<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectionDialog</class>
 <widget class="QDialog" name="SelectionDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>124</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Features by using regular expressions</string>
  </property>
  <property name="toolTip">
   <string>Use a regular expression to make a selection of genes</string>
  </property>
  <property name="statusTip">
   <string>Use a regular expression to make a selection of genes</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widgetBody" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widgetText" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="labelRegExp">
           <property name="text">
            <string>Regular Expression:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editRegExp"/>
         </item>
         <item>
          <widget class="QLabel" name="labelStatus">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="visible">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Invalid!</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widgetOptions" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="checkSelectNonVisible">
           <property name="text">
            <string>Selec Non visibleGenes</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkCaseSense">
           <property name="text">
            <string>Case Sensitive</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SelectionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SelectionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkCaseSense</sender>
   <signal>toggled(bool)</signal>
   <receiver>SelectionDialog</receiver>
   <slot>slotCaseSensitive(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editRegExp</sender>
   <signal>textChanged(QString)</signal>
   <receiver>SelectionDialog</receiver>
   <slot>slotValidateRegExp(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SelectionDialog</sender>
   <signal>signalValidRegExp(bool)</signal>
   <receiver>SelectionDialog</receiver>
   <slot>slotEnableAcceptAction(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SelectionDialog</sender>
   <signal>signalValidRegExp(bool)</signal>
   <receiver>labelStatus</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>390</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkSelectNonVisible</sender>
   <signal>toggled(bool)</signal>
   <receiver>SelectionDialog</receiver>
   <slot>slotSelectNonVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>signalValidRegExp(bool)</signal>
  <slot>slotCaseSensitive(bool)</slot>
  <slot>slotIncludeAmbiguous(bool)</slot>
  <slot>slotValidateRegExp(QString)</slot>
  <slot>slotEnableAcceptAction(bool)</slot>
  <slot>slotSelectNonVisible(bool)</slot>
 </slots>
</ui>
